Storage/DataManagerInterface.php

```diff
@@ -29,6 +29,13 @@ function getStorage();
 	function save(FormFlowInterface $flow, array $data);
 
 	/**
+	 * Checks if a flow exists.
```

Comment:
It's not about the flow itself, but its data. So `Checks if data exists for the given flow.` would fit better.